bgpd/bgp_vty.c

```diff
@@ -7463,11 +7463,14 @@ DEFUN (show_bgp_vrfs,
 			continue;
 
 		count++;
-		if (!uj && count == 1)
+		if (!uj && count == 1) {
 			vty_out(vty,
-				"%4s  %-5s  %-16s  %9s  %10s  %-37s %-10s %-15s\n",
+				"%4s  %-5s  %-16s  %9s  %10s  %-37s\n",
```

Comment:
From the display that you had above, I think numbers under #PeersVfg can be center aligned a bit.

Reply:
Changed the output.